release-notes.md

```diff
@@ -3,6 +3,18 @@ title: Release Notes
 ---
 {% include toc.liquid.md %}
 
+## Release 6.18.0 -- 2019/04/11
+
+### New Features
+
+- Extraction.Web
+  - Inner text excludes script/style tag contents.
+
+### Bug Fixes / Enhancements
+
+- Compound.Split
+  - Improved learning of quoted strings. [Todo]
```

Comment:
What's this [ToDo] about?  Seems like it should be removed.

Reply:
ah, sorry. I meant to delete them.